habitat/core/batched_env.py

```diff
@@ -167,6 +187,9 @@ def __init__(
         include_depth = "DEPTH_SENSOR" in config.SENSORS
         include_rgb = "RGB_SENSOR" in config.SENSORS
 
+        global MAX_EPISODE_LENGTH
```

Comment:
Welp, this has nasty code smell.

Reply:
I agree, we should always pass this around via the config. If a sensor needs it, it should be passed via the config.

Reply:
It does, I am still hacking around.

Reply:
Ok, fixed. @ASzot @Skylion007